### DISCUSSION TITLE
How does the antiSMASH6 determine RiPP precursor

### DISCUSSION THREAD

**Discussion by ZhangDengwei** [antismash/antismash#568] [Q&A]
Hi,

Thanks for the ammazing tool. I wonder how the antiSMASH 6 determines the precursor of RiPP BGCs? I have reviewed the previous antiSAMSH paper and found it seems to use RODEO to locate putative precursors, as described in [antiSMASH 4](https://academic.oup.com/nar/article/45/W1/W36/3778252?login=false). And, if so, is RODEO 1 or 2 adopted?

**Comment by SJShaw** [comment]
RODEO 1 was originally written in perl, the version we use is a python reimplementation of that with some fixes. RODEO 2 was rewritten in python, using much of the same core, but added some extras. So the RODEO version antiSMASH is using is approximately 1 and half.
